lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -296,6 +295,33 @@ describe('NxCombobox', function() {
         expect(onSearch).not.toHaveBeenCalled();
       });
 
+  it('places tooltips on each dropdown item if itemTooltip is defined', async function() {
+    const user = userEvent.setup(),
+        view = quickRender({
+          itemTooltip: ({ displayName }) => `${displayName} tooltipp`,
+          matches: [{ id: '1', displayName: 'Foo' }]
+        }),
+        btn = view.getByRole('option');
+
+    await user.hover(btn);
+    const tooltip = await screen.findByRole('tooltip');
+
+    expect(tooltip).toHaveTextContent('Foo tooltipp');
+  });
+
+  it('does not put tooltips on the dropdown items if itemTooltip is not defined', async function() {
+    const user = userEvent.setup(),
+        view = quickRender({
+          matches: [{ id: '1', displayName: 'Foo' }]
+        }),
+        btn = view.getByRole('option');
+
+    await user.hover(btn);
+    const tooltipPromise = screen.findByRole('tooltip');
+
+    await expect(tooltipPromise).rejects.toBeTruthy();
```

Comment:
I might think it too much..but is it possible that more than one tooltip is found which will make the `tooltipPromise` rejected, and it still passed on line 322?

Reply:
> I might think it too much..but is it possible that more than one tooltip is found which will make the tooltipPromise rejected, and it still passed on line 322?

Nice catch. 00f39d3c5